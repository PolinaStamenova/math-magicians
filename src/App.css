nav {
  padding: 2rem;
  display: flex;
  justify-content: space-between;
  align-items: baseline;
  background-color: grey;
}

nav ul {
  width: 200px;
  display: flex;
  justify-content: space-between;
}
