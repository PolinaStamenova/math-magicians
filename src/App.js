import React from "react";
import Calculator from "./components/Calculator";
import "./components/Calculator.css";

class App extends React.Component {
  render() {
    return <Calculator />;
  }
}

export default App;
