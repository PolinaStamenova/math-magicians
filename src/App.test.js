import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

it("expect quote page to match working snapshot", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
