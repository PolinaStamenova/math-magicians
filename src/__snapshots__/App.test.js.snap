// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote page to match working snapshot 1`] = `
<div
  computedMatch={
    Object {
      "isExact": true,
      "params": Object {},
      "path": "/",
      "url": "/",
    }
  }
  location={
    Object {
      "hash": "",
      "pathname": "/",
      "search": "",
      "state": undefined,
    }
  }
>
  <div>
    <nav>
      <h1
        className="title"
      >
        Math Magicians
      </h1>
      <ul>
        <a
          href="/"
          onClick={[Function]}
        >
          <li
            style={
              Object {
                "color": "blue",
              }
            }
          >
            Home
          </li>
        </a>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          <li
            className="middle"
            style={
              Object {
                "color": "blue",
              }
            }
          >
            Calculator
          </li>
        </a>
        <a
          href="/quote"
          onClick={[Function]}
        >
          <li
            style={
              Object {
                "color": "blue",
              }
            }
          >
            Quote
          </li>
        </a>
      </ul>
    </nav>
  </div>
  <div
    className="home-holder"
  >
    <h2
      className="home-title"
    >
      Welcome to my page!
    </h2>
    <p
      className="home-text"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi.
    </p>
    <p
      className="home-second-text"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi.Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam quis deserunt ut. Expedita delectus, ipsam aperiam iure optio hic praesentium modi repellat in nam ut soluta dolore ex illum nisi. Lorem ipsum dolor sit amet consectetur adipisicing elit.
    </p>
  </div>
</div>
`;
